docs/FilePropertyManagement.md

```diff
@@ -2,37 +2,37 @@
 
-zAppBuild's implementation supports overriding the majority of build properties. The full list can be viewed at [application-conf/README.md](../samples/application-conf/README.md).
+The zAppBuild framework supports overriding the majority of DBB build properties. The full list can be viewed at [application-conf/README.md](../samples/application-conf/README.md).
 
 zAppBuild leverages DBB's API and allows you to define build parameters on three different levels for each language script:
 
-  1. General defaults in corresponding the language property files - for example, defining the compile options for building COBOL programs in [application-conf/Cobol.properties](../samples/application-conf/Cobol.properties). Property keys make use of a language prefix, for instance for COBOL programs this is `cobol_`.
-  2. A group definition, to overriding the default by specifying using DBB's file property syntax using a pattern filter; or through a mapping to a language definition file to override the defaults, like [application-conf/languageDefinitionMapping.properties](../samples/MortgageApplication/application-conf/languageDefinitionMapping.properties). 
-  3. An individual file-level definition to override a single parameter leveraging Dependency Based Builds file properties syntax using [application-conf/file.properties](../samples/application-conf/file.properties) or multiple parameters for a specific file using the individual property file, like [epsmlist.cbl.properties](../samples/MortgageApplication/properties/epsmlist.cbl.properties).
+1. General defaults in the corresponding language property files: For example, you can define the compile options for building COBOL programs in [application-conf/Cobol.properties](../samples/application-conf/Cobol.properties). Property keys make use of a language prefix; for instance, COBOL property keys are prefixed with `cobol_`.
+2. A group definition that overrides the default in one of two ways:
+   - By using DBB's file property syntax and specifying the application artifact group via a pattern filter on the path name(s)
+   - By mapping to a language definition file to override the defaults, such as in [application-conf/languageDefinitionMapping.properties](../samples/MortgageApplication/application-conf/languageDefinitionMapping.properties)
+3. An individual file-level definition to override a single parameter via Dependency Based Build's file properties syntax using [application-conf/file.properties](../samples/application-conf/file.properties) or multiple parameters for a specific file using the individual property file. For example: [epsmlist.cbl.properties](../samples/MortgageApplication/properties/epsmlist.cbl.properties).
```

Comment:
I agree with the way you are suggesting to present the third bullet. Should we also provide some more differentiation between the DBB file property syntax mentioned in the first sub-bullet of 2, versus the first sub-bullet of 3? They sound quite similar to me, except the former (in 2) is being done in `build.properties`, whereas the latter (in 3) is being done in `application-conf/file.properties`. Is my understanding correct?

Reply:
No, the variants described in sub-bullets 2.1 and 3.1 should both be managed in `application-conf/file.properties`. 

It is the same thing:

```properties

# is a group filter, because of the usage of a wildcard
cobol_compileParms=LIB :: MortgageApplication/cobol/EPS*.cbl 

# uses the qualified path of the files
cobol_compileParms=LIB :: MortgageApplication/cobol/EPSCMORT.cbl, MortgageApplication/cobol/EPSCLIST.cbl
```

Reply:
In the IBM Docs that we linked to for the [DBB file property syntax](https://www.ibm.com/docs/en/dbb/2.0.0?topic=apis-build-properties#file-properties), it mentions using this syntax in `build.properties`, which confused me into thinking that 2.1 was being handled there.

However, after rereading this information, my understanding now is that the build properties defined in `build.properties` would actually fall under the first bullet (general defaults).

Perhaps we can clarify 2.1 by specifying the `application-conf/file.properties` location for this bullet, as well.